### DISCUSSION TITLE
OAuth token, Scopes

### DISCUSSION THREAD

**Discussion by HugoB07** [JohnnyCrazy/SpotifyAPI-NET#753] [Q&A]
Hello,

how can I authorize my OAuth token to access scopes ?

my app is in C#, winform, framework : .NET Framwork

Please help me, and thanks

**Comment by JohnnyCrazy** [comment]
Hi, 

The scopes are part of the authentication flows and depend on which kind of flow you use.

Please have a look at the following documentation pages: https://johnnycrazy.github.io/SpotifyAPI-NET/docs/auth_introduction
